test/js/node/http/node-http.test.ts

```diff
@@ -158,6 +158,23 @@ describe("node:http", () => {
       server.close();
     });
 
+    it("should use the provided port", async () => {
+      const server = http.createServer(() => {});
+      server.listen(3000);
```

Comment:
To help prevent flaky tests, instead of a hard-coded port, could you import `randomPort()` from `harness`?

Reply:
sure thing!